kolibri/plugins/learn/assets/src/views/content-points/index.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+
+  <div>
+    <transition name="popup">
+      <div v-if="popoverShown" class="popover-container">
+        <div class="popover">
+          <div class="content">
+            <div class="topline">
+              <points-icon class="icon" :active="true" :width="'20px'" :height="'20px'"/>
+              <span class="plus-points">{{ $tr('plusPoints', { maxPoints }) }}</span>
+            </div>
+            <span class="encourage">{{ $tr('niceWork') }}</span>
+          </div>
+          <ui-close-button
+              size="small"
+              @click="closePopover"
+              class="close"
+          ></ui-close-button>
+        </div>
+      </div>
+    </transition>
+    <div class="points" :style="style">
+      <points-icon class="in-points" :active="active" :width="'30px'" :height="'30px'"/>
+      <span class="count in-points">{{ $formatNumber(maxPoints) }}</span>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  const { contentPoints } = require('kolibri.coreVue.vuex.getters');
+  const { MaxPointsPerContent } = require('kolibri.coreVue.vuex.constants');
+
+  module.exports = {
+    $trNameSpace: 'contentPoints',
+    $trs: {
+      plusPoints: '+ { maxPoints, number } Points',
+      niceWork: 'Nice work. Keep it up!',
```

Comment:
Not super important but just FYI - there was discussion of having a few interchangeable "nice work" messages that would be selected at random